source/jormungandr/jormungandr/interfaces/v1/Disruptions.py

```diff
@@ -120,6 +127,16 @@ def get(self, region=None, lon=None, lat=None, uri=None):
 
         args["filter"] = self.get_filter(split_uri(uri), args)
 
+        since = args.get('since')
+        until = args.get('until')
+        if since and until and since > until:
+            abort(400, message='until must be > since')
```

Comment:
```suggestion
            abort(400, message='until must be >= since')
```
To be coherent with the test above `since > until`

Reply:
OK

Reply:
Done